api/managementpb/dbaas/components.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package dbaas.v1beta1;
+
+option go_package = "api/managementpb/dbaas;dbaasv1beta1";
+
+import "google/api/annotations.proto";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// Component contains information about component.
+message Component {
+  string image_path = 1;
+  string image_hash = 2;
+  string status = 3;
+  bool critical = 4;
+  bool default = 5;
+}
+
+// Matrix contains all available components.
+message Matrix {
+  map<string, Component> mongod = 1;
```

Comment:
Is it `mongod` or `mongodb`?

Reply:
It's `mongod`. That's what we get from the versions service ([here](https://check.percona.com/versions/v1/pxc-operator/1.7.0/8.0.21-12.1)) 🤷‍♂️